src/clj/game/cards/identities.clj

```diff
@@ -1081,9 +1078,9 @@
 (defcard "Jinteki: Potential Unleashed"
   {:events [{:async true
              :event :pre-resolve-damage
-             :req (req (and (-> @state :corp :disable-id not)
-                            (= target :net)
-                            (pos? (last targets))))
+             :req (req (and ;;(-> @state :corp :disable-id not)
```

Comment:
is this the only place that uses `disable-id`? i can't find it elsewhere. we should remove `:disable-id` from all of the cdefs as it's just cruft.

Reply:
I think it might be have been the only one, so I guess I can remove `:disable-id true` from direct access, employee strike and cerebral static and see if anything breaks.